packages/es-components/src/components/patterns/datepicker/DatePicker.js

```diff
@@ -50,7 +49,7 @@ export const DatePicker = props => {
       maskType="date"
       prependIconName="calendar"
       additionalHelpContent={additionalHelpContent}
-      validationState={validationState}
+      {...otherProps}
```

Comment:
It does feel heavy handed.  TextBox passes through a bunch of props and I could see us wanting to add more that pass through this level.  I also agree that we have opened ourselves up for side effects... with the particular props I am passing through there shouldn't really be a problem but I see potential for issues.  At the moment I don't foresee any specific additional props I will need (besides required and validationState) so I could just add required explicitly and move forward.  
Since this was the second instance of me having to add an explicit prop to pass through it seemed like it would make sense  to just pass through additional props, but I'm open to either option.

Reply:
@Darrken What are your thoughts? I'm partially fine with letting this change through, though only if we address this quickly in a future PR, perhaps at the same time as updating `react-date-picker`. I'd feel just a little bit better about specifically passing required through, but it doesn't change the fact that we need to start thinking of a way of exposing `DateTextbox` to be more easily customizable.

Reply:
Required is one of those normal HTML5 attributes which I have been leaning toward leaving out of explicit props along with value, onClick, etc. It helps us support regular attributes people would expect to use without having to document every single one.

With validationState I could go either way. As long as the consumer knows to use it with the datePicker or textbox, I don't mind if it gets lumped into an "otherProps" kind of thing behind the scenes.

Reply:
I agree with that. The reason I am not liking it is that we are passing `otherProps` to both the `DateTextbox` and `ReactDatePicker` components.

Reply:
Oh, I see now. Yeah we can/should address this later with another approach. I was also debating whether we should try using a render prop for the datepicker. We could try this in my dependency-update PR where I have updated the react-datepicker version.

Reply:
So where does this leave us? 
Do I need to make an update here?